unitTests/ui/DidYouMeanTest.ts

```diff
@@ -57,6 +57,24 @@ export function DidYouMeanTest() {
       expect(test.cmp.element.style.display).toBe('none');
     });
 
+    it("should add a href to the correction's link when there's a result", () => {
+      Simulate.query(test.env, {
+        results: FakeResults.createFakeResults(1),
+        queryCorrections: [fakeQueryCorrection]
+      });
+
+      expect(test.cmp.element.querySelector('a[href="javascript:void(0);"]')).not.toBeNull();
+    });
+
+    it("should add a tabindex to the correction's link when there's a result", () => {
+      Simulate.query(test.env, {
+        results: FakeResults.createFakeResults(1),
+        queryCorrections: [fakeQueryCorrection]
+      });
+
+      expect(test.cmp.element.querySelector('a[tabindex]')).not.toBeNull();
```

Comment:
I would prefer the stricter check that the value is `0` here. If the attribute was set to `-1`, this test would pass but the behaviour would not be correct.

Reply:
Very good point. Those unit tests won't be necessary anymore though, since I replaced the anchor with a button.